scripts/devenv-builder/configure-vm.sh

```diff
@@ -47,6 +47,20 @@ if [ $RHEL_SUBSCRIPTION = true ] ; then
     fi
 fi
 
+# Temporarily use RHEL 9.2 nightly repositories
+sudo tee /etc/yum.repos.d/rhel-92.repo >/dev/null <<EOF
+[rhel-9-for-x86_64-appstream-nightly-rpms]
+name = Red Hat Enterprise Linux 9 for x86_64 - AppStream Nightly (RPMs)
+baseurl = http://download.eng.tlv.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.2.0/compose/AppStream/x86_64/os/
```

Comment:
This is a region-specifc URL. Is there a location that redirects to something local to the user?